README.md

```diff
@@ -369,7 +370,57 @@ Another example
 t.criteo :track_transaction, { id: 'id', item: { id: "P0038", price: "6.54", quantity: 1 } }
 ```
 
+### Zanox
+
+[Zanox](http://www.zanox.com/us/)
+
+#### Basic Configuration
+
+```
+config.middleware.use(Rack::Tracker) do
+  handler :zanox, { account_id: '1234' }
 end
+```
+
+#### Mastertag
+
+This is an example of a mastertag:
+
+```
+def show
+  tracker do |t|
+    t.zanox :mastertag, { id: "25GHTE9A07DF67DFG90T" }
+  end
+end
+```
+
+This will render to the follwing code in the JS:
+
+```
+window._zx.push({"id": "25GHTE9A07DF67DFG90T"});
+```
+
+#### Conversion tracking
+
+This is an example of a lead event:
+
+```
+def show
+  tracker do |t|
+    t.zanox :track, { order_i_d: 'DEFC-4321'}
```

Comment:
I think it's because we use camelize on those (https://github.com/railslove/rack-tracker/pull/37/files#diff-274286769fdcfc86290f48b5899263c2R12)